packages/client/src/App.js

```diff
@@ -1,17 +1,16 @@
-/* eslint-disable */
+// /* eslint-disable */
 import React from 'react';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import './App.css';
-import { LandingPageContainer } from './containers/LandingPage/LandingPage';
-import { Dashboard } from './containers/LandingPage/Dashboard';
+import Dashboard from './containers/Dashboard/Dashboard';
+// import { LandingPageContainer } from './containers/LandingPage/LandingPage';
```

Comment:
🗑️

Reply:
at the moment I have to keep the comment because the Navbar has not finished, So I can keep working. if I comment out the page will be broken and also we have the landing page design as well so we need this in app.js. For now I am waiting for the Navbar PR to be solved.